Add missing LDAP integration properties (#139)
exoplatform/exo-documentation: docs/LDAP.rst

This section is a complete description of the available configuration options. 
It lists the options of both eXo configuration and PicketLink configuration.

.. _Ref_properties:

Properties configuration
------------------

> The LDAP integration uses jobs to synchronize periodically modified data on LDAP into eXo internal database.

Reviewer: ```suggestion
The LDAP integration uses jobs to synchronize periodically the eXo internal database with the data modified in the LDAP.
```

Author: applied

> The list of properties that you can use in ``exo.properties`` are:

Reviewer: ```suggestion
The list of properties related to LDAP integration that you can use in ``exo.properties`` are:
```

Author: applied

.. code-block:: jproperties

    # Cron expression used to schedule the job that will import periodically data
    # from external store
    # (Default value = every ten minutes)
    exo.idm.externalStore.import.cronExpression=0 */10 * ? * *
    # Cron expression used to schedule the job that will delete periodically data
    # from internal store that has been deleted from external store
    # (Default value = every day at 23:59 PM)
    exo.idm.externalStore.delete.cronExpression=0 59 23 ? * *
    # Cron expression used to schedule the job that will process periodically data
    # injected in queue
    # (Default value = every minute)
    exo.idm.externalStore.queue.processing.cronExpression=0 */1 * ? * *

> By default, user data are updated when the user login. This can be disabled by modifying this property:

Reviewer: ```suggestion
By default, user data are updated when the user signs in. This can be disabled by modifying this property:
```

Author: applied

> By default, the user data are updated when the user login. This can be disabled by modifying this property:

Reviewer 2: ```suggestion
By default, user data are updated when the user login. This can be disabled by modifying this property:
```

Author: done

> By default, user data are updated when the user signs in. This can be disabled by modifying this property:

Reviewer: ```suggestion
By default, user data are synced with the LDAP when the user signs in. This can be disabled by modifying this property:
```

.. code-block:: jproperties

>     # if true, update user data on login time and only when the information changes

Reviewer 2: ```suggestion
    # if true, update user data on login time and only when information change
```

Author: done

    # on external store (Default: true)
    exo.idm.externalStore.update.onlogin=true

> The user data import tasks are made asynchronously, thus a queue is used to process those tasks and a maximum number of synchronization operation retries can be configured by changing the following property:

Reviewer 2: ```suggestion
The user data import tasks are made asynchronously, thus a queue is used to process those tasks and a maximum number of synchronization operations retrials can be configured by changing the following property:
```

Author: are you sure that it's not `retries` instead:
https://www.linguee.fr/anglais-francais/traduction/number+of+retries.html
VS
https://www.linguee.fr/anglais-francais/traduction/retrial.html

In fact, it one operation that is retried multiple time and not multiple operations retried multiple times. That's why it singular `operation`

Reviewer 2: AH ok I misunderstood the signification

> The user data import tasks are made asynchronously, thus a queue is used to process those tasks and a maximum number of synchronization operation retries can be configured by changing the following property:

Reviewer: ```suggestion
The user data sync tasks (one per user) are executed asynchronously via a queue. If a sync task fails, it can be retried. The maximum number of sync retries before the task is abandoned in error can be configured by changing the following property:
```
